go.mod

```diff
@@ -36,6 +36,12 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
+	github.com/openshift/apiserver-library-go v0.0.0-20250502092109-8d341e94467d
+	k8s.io/apiserver v0.32.3
+	k8s.io/kubernetes v1.33.0
+)
```

Comment:
why a separate group?

Reply:
Mistake probably. will fix in next iteration